### DISCUSSION TITLE
Can I use axios to send http request?

### DISCUSSION THREAD

**Discussion by Virgil-N** [wailsapp/wails#820] [Q&A]
I'm trying to use axios to send a http request, but failed. 
Error shows I don't have permission.

**Comment by leaanthony** [comment]
Hmm.. not sure. OS? Screenshot?

**Reply by Virgil-N** [reply]
Windows 10
go 1.17
wails 1.6.6
No pictures. Error shows access denied. 
PS: I'm using axios with react.

**Reply by leaanthony** [reply]
I'm not sure without the error.

**Reply by Virgil-N** [reply]
Seems I can't use redux

**Reply by leaanthony** [reply]
I'm sure you can. If you have any more details - let us know.

**Reply by Virgil-N** [reply]
I tried with go to send http request, but got error code 400. Seems request was not sent, because there is no information print in backend.

**Reply by leaanthony** [reply]
Sounds like a problem with the service you're trying to access. If you find this is a bug related to UI then please open a ticket.

**Comment by raguay** [comment]
I'm using fetch with a second local server (not the wails server) without any issues. Doesn't Axios use fetch inside?
